lib/config-example.js

```diff
@@ -0,0 +1,89 @@
+/**
```

Comment:
It's good to show examples for stuff, but this is kind of over the top. I'd just remove this file entirely and instead write some more comments in the config.js file instead.

Reply:
@landonreed and I started with all the comments in `config.js` and found the file to become very cluttered.

Reply:
The config.js file is an example file itself. It is good enough. This file is not needed.